common/types/src/runtime.proto

```diff
@@ -0,0 +1,118 @@
+syntax = "proto3";
+
+package kyve.tendermint;
+
+import "google/protobuf/empty.proto";
+
+
+message DataItem {
+    string key = 1;
+    google.protobuf.Any value = 2;
+}
+
+message RuntimeConfig {
+  // Define fields for the runtime configuration
+  // ...
+}
+
+/* getName */
+message GetNameRequest {
```

Comment:
Can we be more explicit on what `GetNameRequest` is? Which name does it request?

Reply:
would `getRuntimeName` make more sense?